### ISSUE TITLE
Hidden Antiquity Sites Causing Diplomatic Repercussions from Non-artistry civs

### ISSUE THREAD

**Issue by deadstarre** [LoneGazebo/Community-Patch-DLL#4033]
_1. Mod version (i.e Date - 4/23):_

2/7 

_2. Mod list (if using Vox Populi only, leave blank):_

_3. Error description:_ 

_4. Steps to reproduce (optional):_

Take Artistry, dig a 'hidden' antiquity site on India property (mutually open borders), and although he took Statecraft so he cannot see this site and would never be able to get it, he is upset that I dug it up, I get negative diplo modifier and he wants a promise for me not to do it again. is this Intentional or bug? if intentional, it doesn't seem to make any sense, unless this is purely a balance thing and Artistry would be too powerful otherwise.
---------------------------
Supporting information:
Please note that you can attach .zip files by dragging-and-dropping them. If possible, zip up all supporting data and post that way.

1. Log files (always attach your Logs folder, located at My Documents/My Games/Sid Meier's Civilization 5. Make sure you have enabled logging before experiencing an error! Go here to find out how: http://forums.civfanatics.com/showthread.php?t=487482):

2. Save game (always attach a save that was made a turn before the error; located at My Documents/My Games/Sid Meier's Civilization 5/ModdedSaves):

3. CvMiniDump.dmp file (attach if experiencing a game crash. Located at Program Files/Steam/steamapps/common/Sid Meier's Civilization V):

4. Screenshots (optional):

**Comment by LoneGazebo** [comment]
@deadstarre it's intended, as a human player would 100% know what the AI is doing if an archaeologist showed up and started digging on 'nothing.' I put it in because a late grab of artistry (if few others have gone for it) is not uncommon.

**Comment by deadstarre** [comment]
well, if it was a normal site yeah i'd know what they were doing and be upset, because the tech is still coming to me. if I knew it was a special site, exclusive to them and something i won't be getting regardless, and im on friendly enough terms to exchange open borders... I wouldnt be upset at all. sure a late grab of artistry is always a possibility, but considering you need to fill out the entire tree to see them thats a pretty significant investment... I mean I know people (not AI) go back to cherry pick policies, but cherry picking the entire artistry tree... I dunno about that honestly.  

anyway, my mother insists I bring this up as well - shes playing a game and Austria paid her to go to war with her neighbor, except Austria hates warmongers and so she wound up with a negative modifier (and ultimately lost the friendship) for going to war with Austria's enemy.  She wants to put in "in real life its never a negative with your allies or peoples you have pacts with you would never consider them a warmonger for helping to defend you in your cause. "  I told her warmonger is lessened for friends, but she insists the penalty should have been zero.  I had to hear it, now you feel the wrath too.

**Comment by bigcat88** [comment]
I agree with deadstarre. There must not be negative diplomodifier for digging hidden antiquity in civ lands that are not go artistry.
And when I build one landmark in friend civ's land and want to build second landmark - why they ask me to stop digging?

**Comment by deadstarre** [comment]
the other thing I remembered is that mutually open borders was not always a requirement to dig. then I wondered if the diplo hit was added before or after that came into effect - because the open border req already tames digging in foreign lands quite a bit, especially where late game grabs in a post-ideology world are of concern

**Comment by LoneGazebo** [comment]
I think you are asking the AI to be less shrewd. Not sure I agree.

**Comment by deadstarre** [comment]
full disclosure this was my mothers issue as well lmao but she makes sense to me. just put yourself in AI shoes a moment and think it through. 

You are AI, you took a different medieval tree than artistry. So now we know you will never take artistry (except rarely when taking 2x medieval trees?). So there is your friend Persia, you two share open borders because youre on good terms. He come in your territory (as you allow) and dig himself up a bonus that he has earned which actually has nothing to do with you, and won't affect you negatively in any way shape or form, ever. so would you really be mad about it? why? if that makes you mad, shouldn't you also be just as likely to get mad at him for earning himself golden ages? its just another bonus that he earned for himself thats literally nothing to do with you.   

I see this as it would be more reasonable behavior by AI, not less shrewd, but I made my case. thanks for the attention ;)
